sdk-springboot/pom.xml

```diff
@@ -0,0 +1,64 @@
+<project
+    xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>io.dapr</groupId>
+    <artifactId>dapr-sdk-parent</artifactId>
+    <version>0.2.0</version>
+  </parent>
+
+  <groupId>io.dapr</groupId>
+  <artifactId>dapr-sdk-springboot</artifactId>
+  <packaging>jar</packaging>
+  <version>0.2.0</version>
+  <name>dapr-sdk-springboot</name>
+
+  <properties>
+    <java.version>8</java.version>
+    <maven.compiler.source>${java.version}</maven.compiler.source>
+    <maven.compiler.target>${java.version}</maven.compiler.target>
+  </properties>
+
+  <dependencies>
+    <dependency>
+      <groupId>javax.annotation</groupId>
+      <artifactId>javax.annotation-api</artifactId>
+      <version>1.3.2</version>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot</artifactId>
+      <version>2.2.2.RELEASE</version>
+    </dependency>
+    <dependency>
+      <groupId>org.springframework.boot</groupId>
+      <artifactId>spring-boot-starter-web</artifactId>
+      <version>2.2.2.RELEASE</version>
```

Comment:
You should probably externalise boot versions, but a better approach would probably be to have parent as
```
<parent>
	<groupId>org.springframework.boot</groupId>
	<artifactId>spring-boot-starter-parent</artifactId>
	<version>2.2.2.RELEASE</version>
	<relativePath /> <!-- lookup parent from repository -->
</parent>
```
as pretty much all boot-based projects have. In fact you can go to https://start.spring.io/ and generate a simplest project and us it's pom as a guide on what this one should be

Reply:
We wanted to keep our jars under the same parent and let the user application be a child of SpringBoot. So, I went through the examples and eventually figured out a way to accomplish this.

What do you mean by externalize boot version?